### PR TITLE
Fix: Organization Interface Display Issue

### PR DESCRIPTION
**Pull request by HEREYUA**
**Before**

image

**After**

image

### CODE REVIEW

**Review by lunny — APPROVED** [review]

**Review by silverwind — CHANGES_REQUESTED** [review]
Sorry but that isn't right. There should only be one `.page-content` on the page, with this there would be two. `secondary-nav` is meant for a nav below the main nav, but in this case it does not fit because there is the org header above.

**Comment by silverwind** [comment]
I think this needs some margins added, ideally so that these two divider lines align:

image

**Comment by wxiaoguang** [comment]
Agree, it is not right. `page-content` is quite special, it means the "main content of a page", it shouldn't be abused anywhere else.

**Comment by HEREYUA** [comment]
> I think this needs some margins added, ideally so that these two divider lines align:

Updated, The dividing lines cannot be perfectly aligned unless more detailed fine-tuning is done
image
image

**Comment by silverwind** [comment]
I think to properly fix the line alignment those elements need to be put into the same `<div>`, but that's something for another PR.

File: templates/org/home.tmpl [hunk 1]
```diff
@@ -16,7 +16,7 @@
 			{{if .ShowMemberAndTeamTab}}
 			<div class="ui five wide column">
 				{{if .CanCreateOrgRepo}}
-					<div class="center aligned">
+					<div class="center aligned tw-mb-4">
```
Comment (silverwind) [inline templates/org/home.tmpl:19]:
```suggestion
					<div class="center aligned">
```

Change does not seem to do anything useful regarding the problem.

Comment (silverwind) [reply templates/org/home.tmpl:19]: Actually no, I see what it does. It's ok.

**Review by silverwind — APPROVED** [review]

File: templates/org/home.tmpl (hunk 1 above)
Comment (HEREYUA) [reply templates/org/home.tmpl:19]: Alright, this is to align as much as possible, I have added it again.

**Review by delvh — APPROVED** [review]

**Pull request merged**